### DISCUSSION TITLE
Adding a leader key binding results in "undefined"

### DISCUSSION THREAD

**Discussion by brabalan** [meow-edit/meow#567] [Q&A] [answered]
Hello,

I’m trying to add a keybinding to easily change a dictionary in a buffer, so I tried this:
```emacs-lisp
(meow-leader-define-key '("r f" . (lambda () (jinx-languages "fr"))))
```

Unfortunately, when I type `SPC r f`, I get an error "r f is undefined", even though I see in the minibuffer that the shortcut is there. I’m sure I’m doing this wrong, but I don’t understand why.

**Comment by DogLooksGood** [comment, marked as answer]
To define a command, you must have `(interactive)` in your function definition.

**Comment by brabalan** [comment]
That was it, thanks. Even if it is not a restriction of meow, it could be nice to state it in the documentation.
